### PR TITLE
feat: adds Sparkbox PR checklist

### PR DESCRIPTION
**Pull request by silverli**
# READY FOR REVIEW

Issue: https://sparkbox.atlassian.net/browse/STN-64

# Summary
Adds a PR checklist for Sparkbox devs, with our browser QA checklist

# Steps to Test
1. Looks over the docs and review for anything that seems off! I worked off of Shea's original PR checklist, and picked out what was relevant to us from it!

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by nrambeck — APPROVED** [review]
Looks good on my end!

File: docs/SparkboxPRChecklist.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+#### Accessibility
+- [ ] Are markup and styles accessible?
+
+#### Security
+- [ ] Do forms need to be sanitized?
+- [ ] Any obvious security flaws or potential holes?
+
+#### Internationalization
+is this happening as of right now?
```
Comment (nrambeck) [inline docs/SparkboxPRChecklist.md]: It's not clear what this line is for?

**Review by sherakama — APPROVED** [review]

File: docs/SparkboxPRChecklist.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+#### Accessibility
+- [ ] Are markup and styles accessible?
+
+#### Security
+- [ ] Do forms need to be sanitized?
+- [ ] Any obvious security flaws or potential holes?
+
+#### Internationalization
+is this happening as of right now? 
+- [ ] Is all content (labels and other strings) translated?
```
Comment (silverli) [inline docs/SparkboxPRChecklist.md]: @sherakama could you clarify how you and SWS are generally applying the `trans` or translate filter to UI text? What cases are important to cover, or what's the reasoning behind the use on different sites?

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr435-9zsebvtphlwekjxucceytqagqttw6z8j.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5df7b34897c702dab8f9ea4a

Visual Diffs:
- https://dashboard.tugboat.qa/5df7b34897c702dab8f9ea4a/visualdiffs

File: docs/SparkboxPRChecklist.md (hunk 2 above)
Comment (sherakama) [reply docs/SparkboxPRChecklist.md]: Copying from our slack channel so we have it documented here as well

> We follow best practices and run all strings through the translation APIs in PHP and JS when we should and can. I suspect we may have missed some as we don't have any multilingual sites but it is part of our core practice to support multilang

> I'd say stick to the Drupal best practice conventions and run strings through t & trans but don't go to the extent of creating tests for it.

**Pull request merged**